scripts/release/version.mjs

```diff
@@ -45,7 +44,8 @@ export async function version_handler(version) {
 		// skip all example upgrade
 		if (
 			workspace.manifest.name?.includes("example-") ||
-			workspace.manifest.private === true
+			(workspace.manifest.private === true &&
```

Comment:
```suggestion
			(workspace.manifest.private &&
```

Reply:
we should use `workspace.manifest.private === true` since other field is invalid